src/cli.rs

```diff
@@ -33,7 +33,23 @@ pub fn main() -> Result<()> {
                 .global(true)
                 .help("Sets a custom root working directory"),
         )
-        .subcommand(SubCommand::with_name("update").about("Update the dependencies"))
+        .arg(
+            Arg::with_name("local")
+                .short("l")
```

Comment:
```suggestion
```
I would prefer not to introduce a global short flag `-l` as alias for `--local`.  `-l` could abbreviate `--list`, `--link`, or similar, so I don't consider it intuitive.

If you agree, please update the mentions of the flag as well.